dtfs-central-api/api-tests/v1/utilisation-reports/post-keying-data.api-test.ts

```diff
@@ -392,12 +392,12 @@ describe(`POST ${BASE_URL}`, () => {
     expect(entities[0].id).toEqual(facilityId);
     /**
      * The fixed fee is calculated as follows:
-     * fixed fee = utilisation * bank fee adjustment * interest percentage * days left in cover period / day count basis
-     *           = 1 * 0.9 * 100 / 100 * 730 / 1
-     *           = 0.9 * 730
+     * fixed fee = (utilisation * (coverPercentage / 100)) * bank fee adjustment * interest percentage * days left in cover period / day count basis
+     *           = 1 * (80 / 100) *  0.9 * 100 / 100 * 730 / 1
```

Comment:
Please can we set it to 80 in the test set up rather than relying on the value in `aFacility` so this test success/failure is not tied to the specifics of that function?